src/v/model/adl_serde.h

```diff
@@ -24,6 +25,21 @@
 
 namespace reflection {
 
+template<>
+struct adl<model::timeout_clock::duration> {
+    using duration = model::timeout_clock::duration;
+
+    void to(iobuf& out, duration dur) {
+        adl<std::chrono::milliseconds>{}.to(
+          out, std::chrono::duration_cast<std::chrono::milliseconds>(dur));
+    }
```

Comment:
this was serialized by `uint64_t` and now it looks like it is an `int64_t`. is that true? seems problematic if so.

Reply:
It's fine: https://github.com/redpanda-data/redpanda/pull/6206